contracts/src/bridge/L2/L2_Bridge.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity >=0.7.0 <0.9.0;
+
+
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "../IBridge.sol";
+import "../IBridgeSubordinate.sol";
+import "../../messaging/messenger/CrossChainEnabledObscuro.sol";
+import "./ObscuroERC20.sol";
+
+contract ObscuroL2Bridge is IBridge, IBridgeSubordinate, CrossChainEnabledObscuro {
```

Comment:
a better name is "EthereumBridge"
Since this lives on Obscuro

Reply:
I'm not sure how I feel about "Ethereum"; Perhaps I'm nitpicking but I feel it might be misinterpreted as "standard Ethereum bridge" as in not developed by us, but rather the org?

Reply:
what about "BridgeToL1"

Reply:
and we apply this pattern everywhere: 
"BridgeToX"

that should be pretty explicit